### ISSUE TITLE
Accessibility of editor to screenreader

### ISSUE THREAD

**Issue by webczat** [ashmind/SharpLab#480]
TThe sharplab editor is not accessible to screenreaders. I am not sure if this is a separate project or not...
Generally I cannot navigate it/read contents and modify. Screenreader is silent. The only way for me to do anything is to copy the current contents to a standalone editor, modify and copy it back. However it is rather a workaround.

**Comment by ashmind** [comment]
Thank you for reporting. Sadly I am using an external editor that's far too complex for me to fix on my own. However they are working on a new version that should be properly accessible and I am planning to look at providing that option. Initially it will have less editing features (e.g. autocomplete), but eventually I'll fully migrate to it.

I'll let you know once that option is available.
